ci-operator/jobs/maistra/maistra-test-tool/maistra-maistra-test-tool-maistra-2.4-periodics.yaml

```diff
@@ -0,0 +1,76 @@
+periodics:
+- agent: kubernetes
+  cluster: build03
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: maistra-2.4
+    org: maistra
+    repo: maistra-test-tool
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: servicemesh-ocp4.13-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-maistra-maistra-test-tool-maistra-2.4-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws
```

Comment:
We'll need to [set up Slack alerts](https://cspi-qe.gitbook.io/ocp-ci-onboarding/tutorials/reporting_guide#how-to-setup-slack-alerts-for-a-scenario) for this scenario. Just use whichever Slack channel PQE would like the alerts to go to.

Reply:
@FilipB I am using the `ossm-qe` for the slack alerts

Reply:
ok, let's use _ossm-qe_ for now. We might want to change it later but it's good for now